pybrake/git.py

```diff
@@ -48,3 +48,20 @@ def get_git_head(dirpath):
   head_file = os.path.join(dirpath, '.git', 'HEAD')
   with open(head_file) as f:
     return f.read().rstrip()
+
+def find_git_dir(directory):
+  directory = os.path.abspath(directory)
+  if not os.path.exists(directory):
+    return "", False
+
+  for _ in range(1, 10):
+    path = os.path.join(directory, '.git')
+    if os.path.exists(path):
+      return directory, True
+
+    if directory == "/":
+      return "", False
+
+    directory = os.path.abspath(os.path.join(directory, os.pardir))
+
+  return "", False
```

Comment:
return single value:
- empty string string when not .git dir found
- non empty string otherwise